src/test/java/com/endava/internship/mocking/repository/InMemPaymentRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package com.endava.internship.mocking.repository;
+
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import com.endava.internship.mocking.repository.InMemPaymentRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+@ExtendWith(MockitoExtension.class)
+public class InMemPaymentRepositoryTest {
+
+    private InMemPaymentRepository repository;
+
+    private User user1;
+    private User user2;
+    private Payment payment;
+    private Payment payment1;
+
+    @BeforeEach
+    void setUp() {
+        user1 = new User(1, "John", Status.ACTIVE);
+        user2 = new User(2, "Maria", Status.ACTIVE);
+        payment = new Payment(user1.getId(),2000.1,null);
+        payment1 = new Payment(user2.getId(),2000.3,null);
```

Comment:
```suggestion
        payment = new Payment(1,2000.1,null);
        payment1 = new Payment(2,2000.3,null);
```
if you just use Users to get id for payment creation

Reply:
ok